### ISSUE TITLE
Pathfinder Picaroon TWFinesse and Weapon Proficiency

### ISSUE THREAD

**Issue by reesmanp** [PCGen/pcgen#1884]
A fresh download of the nightly build brings up two issues on the Picaroon.

The weapon proficiencies are not mapped correctly, the simple and martial weapons are shown as not proficient. Only the gained weaponry from Swashbuckler->Picaroon is shown as proficient. A fix of:
`CATEGORY=Special Ability|Picaroon ~ Weapon Proficiency.MOD	ABILITY:Internal|AUTOMATIC|TYPE=WeaponProfMartial	AUTO:WEAPONPROF|TYPE=OneHandedFirearm`
Seems to fix it.

When a rapier is equipped on the primary hand and a pistol is equipped on the off-hand, I can tell that the feats of weapon finesse and two weapon fighting are not being applied. My strength is being used (A +2 as opposed to a +5 dex) and there is a 5 point to-hit drop for my pistol.

**Comment by LegacyKing** [comment]
Hi @reesmanp -- you've done all the work, any reason why you don't submit a PR to get officially listed as a contributor?

**Comment by LegacyKing** [comment]
Also, just a side note - two different issues should be two different entries.

**Comment by reesmanp** [comment]
@LegacyKing Thanks for the replies. Google has actually done the grunt work for me on this one, but sure, I'll put in a PR for the first bit.

I combined these issues into one since I think they are quite related, but I can make a new issue out of the second part.

**Comment by LegacyKing** [comment]
I always explain it this way to folks - if you have one issue, then you focus on fixing the one issue, and once done, the issue is finished. However, if I have two issues in the same tracker, then I am forced to focus on both issues before any progress is made. Like your little fix, this issue would be closed. But since you tagged in another issue, we couldn't close this because now we have to address an actual "unrelated" issue, since data may or may not be involved, it could dive into an Output sheet issue. Who knows. Point is, you can't feel satisfied you handled an issue cause there is still something else that needs to be solved. 

Hope that explanation makes sense. Another way to put it, do you want someone to fix the issue quickly, or did you want to wait for the person who has years of experience to fix the issue. Single issues can typically be handled by most anyone, complicated issues (more than one) scare most volunteers, forcing a senior developer to get involved. Sadly, I'm a senior developers, and as you can see, hours pass before I become available to even answer a question. 

Anyways, send the problem character pcg file to help@pcgen.org and I'll take a look at it.

Cheers,
